src/yamlfix/adapters.py

```diff
@@ -104,6 +110,147 @@ def _configure_quotation_for_basic_values(self) -> None:
         if not config.quote_basic_values or config.quote_representation is None:
             return
 
+        def patch_quotations(
+            key_node: ruyaml.Node, value_node: ruyaml.Node  # noqa: W0613
+        ) -> None:
+            if (
+                isinstance(value_node, ruyaml.ScalarNode)
+                and value_node.tag == "tag:yaml.org,2002:str"
+                and value_node.style is None
+            ):
+                value_node.style = config.quote_representation
+
+        self._patch_mapping_representer_value(patch_quotations)
+
+    def _configure_flow_style_for_sequences(self) -> None:
+        """Configure Ruamel's yaml fixer to represent lists as either block-style \
+            or flow-style.
```

Comment:
I'm not familiar with the differences between `block-style` and `flow-style`. Other code readers may not be either, could you explain the differences in this docstring (and in the doc for the users too)